### PR TITLE
BZZ1882647 - clarifying DHCP configuration for vsphere IPI

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1882647

### CODE REVIEW

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 1]
```diff
@@ -71,7 +71,8 @@ Available resources vary between clusters. The number of possible clusters withi
 [id="installation-vsphere-installer-infra-requirements-networking_{context}"]
 == Networking requirements
 
-In addition to using DHCP for the network, you must create the following networking resources before you install the {product-title} cluster:
+You must use DHCP for the network and ensure that the DHCP server is configured to provide persistent IP addresses and host names to the cluster machines.
+Additionally, you must create the following networking resources before you install the {product-title} cluster:
 
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-_{context}"]
```
Comment (jboxman) [inline modules/installation-vsphere-installer-infra-requirements.adoc:78]: @kalexand-rh, looks like there's an extra `-` in the `id`.

**Comment by bobfuru** [comment]
LGTM

**Pull request merged**